### PR TITLE
Fixes Figures URL wildcard placement

### PR DESCRIPTION
**Pull request by johnbaldwin**
The issue was that the router URLs were included after the wildcard url
pattern. This should fix the issue

### CODE REVIEW

File: figures/urls.py [hunk 1]
```diff
@@ -139,3 +139,5 @@
     urlpatterns.append(url(r'^api/', include(router.urls, namespace='api')))
 else:
     urlpatterns.append(url(r'^api/', include((router.urls, 'api'), namespace='api')))
+
+urlpatterns.append(url(r'^(?:.*)/?$', views.figures_home, name='router-catch-all'))
```
Comment (OmarIthawi) [inline figures/urls.py:143]:
```suggestion
# Reroute all unmatched traffic to Figures main UI page
urlpatterns.append(url(r'^(?:.*)/?$', views.figures_home, name='router-catch-all'))
```

File: figures/urls.py [hunk 2]
```diff
@@ -129,7 +129,7 @@
         name='general-site-metrics'),
 
     # Reroute all unmatched traffic to Figures main UI page
-    url(r'^(?:.*)/?$', views.figures_home, name='router-catch-all')
+    # url(r'^(?:.*)/?$', views.figures_home, name='router-catch-all')
```
Comment (OmarIthawi) [inline figures/urls.py:132]:
```suggestion
```

File: figures/urls.py [hunk 3]
```diff
@@ -129,7 +129,7 @@
         name='general-site-metrics'),
 
     # Reroute all unmatched traffic to Figures main UI page
```
Comment (OmarIthawi) [inline figures/urls.py:131]:
```suggestion
```

**Review by OmarIthawi — APPROVED** [review]

**Comment by johnbaldwin** [comment]
Thanks, Omar!

**Review by melvinsoft — APPROVED** [review]
@johnbaldwin Except Omar's comments, it looks good to me.

**Pull request merged**